src/components/Container/Container.scss

```diff
@@ -0,0 +1,117 @@
+@import './../../styles/fonts.scss';
+@import './../../styles/breakpoints.scss';
+@import './../../styles/alignment.scss';
+
+.container {
+  display: grid;
+  text-align: center;
+  max-width: $breakpoint-laptop-large;
+  margin: 0 16px;
+  padding: 5px;
+}
+
+// This column class could be removed entirely and the grid system would work. We could consiser adding the padding and margin to the layout classes, and controlling this in the layout classes.
+
+.column {
+  padding: 10px;
+  margin: 10px;
+}
+
+/* Layouts  */
+
+// 2 Columns
+
+.cols-2 {
+  grid-template-areas: 'column column';
+  grid-template-columns: 1fr, 1fr;
+  grid-auto-flow: column;
+  @media screen and (max-width: $breakpoint-tablet) {
+    grid-template-areas: 'column column';
+    grid-template-columns: 1fr 1fr;
+  }
+  @media screen and (max-width: $breakpoint-mobile) {
+    grid-template-areas:
+      'column'
+      'column';
+    grid-template-columns: 1fr;
+  }
+}
+
+// 3 Columns
+
+.cols-3 {
+  grid-template-areas: 'column column column';
+  grid-template-columns: repeat(3, 1fr);
+  grid-auto-flow: column;
+  @media screen and (max-width: $breakpoint-tablet) {
+    grid-template-areas: 'column column column';
+    grid-template-columns: repeat(3, 1fr);
+  }
+  @media screen and (max-width: $breakpoint-mobile) {
+    grid-template-areas:
+      'column'
+      'column'
+      'column';
+    grid-template-columns: 1fr;
+  }
+}
+
+// 4 Columns
+
+.cols-4 {
+  grid-template-areas: 'column column column column';
+  grid-template-columns: repeat(4, 1fr);
+  @media screen and (max-width: $breakpoint-tablet) {
+    grid-template-areas: 'column column';
+    grid-template-columns: 1fr 1fr;
+  }
+  @media screen and (max-width: $breakpoint-mobile) {
+    grid-template-areas:
+      'column'
+      'column'
+      'column'
+      'column';
+    grid-template-columns: 1fr;
+  }
+}
+
+// 5 Columns
+
+.cols-5 {
+  grid-template-areas: 'column column column column column';
+  grid-template-columns: repeat(5, 1fr);
+  @media screen and (max-width: $breakpoint-tablet) {
+    grid-template-areas: 'column column column column column';
+    grid-template-columns: repeat(5, 1fr);
```

Comment:
are the top and bottom grid-template-columns suppose to be the same? is the bottom needed?

Reply:
@mcheng09 they were, but we want the total amount of columns to be divided in 1/2 for tablet layouts.

That line above should have been `repeat(3, 1fr)`